revolv/static/dashboard.scss

```diff
@@ -72,3 +91,116 @@
     max-width: 100%;
     float: left;
 }
+.project-container {
+    padding-left: 0px;
+    padding-right: 0px;
+    padding-top: 0px;
+    padding-bottom: 0px;
+    margin: 0px, 0px, 0px, 0px;
+
+    .status-bar {
+    transition: border-color 200ms linear;
+    transition: background-color 200ms linear;
+    height: 40px;
+    width: 80%;
+    position: absolute;
+    top: 50%;
+    left: 10%;
+    border-color: white;
+    border-width: 2px;
+    border-style: solid;
+
+        .status-bar-menu-text {
+            transition: color 200ms linear;
+            font-family: $body-font-family;
+            text-align: center;
+            top: 50%;
+            margin-right: 6px;
+            color: white;
+            font-weight: 300;
+            z-index: -1;
+        }
+
+        &.active, &:hover {
+            border-width: 2px;
+            border-style: solid; 
+
+            .status-bar-menu-text {
+                color: $revolv-blue;
+            }
+        }
+
+        .progress-bar-scale {
+            height: 10%;
+            width: 100%;
+            background-color: white;
+            position: absolute;
+            top: 90%;
+            bottom: 10%;
+
+            .green-bar {
+                width: 90%;
+                height: 100%;
+                background-color: green;
+            }
+
+        }
+    }
+
+    img {
```

Comment:
Would be best to use a class here instead of the `img` tag name, there may be images on the page that we don't want to be position: relative